Dockerfile

```diff
@@ -5,6 +5,7 @@ ENV UPLOAD_MAX_FILESIZE=24M
 ENV POST_MAX_SIZE=24M
 ENV MAX_FILE_UPLOADS=201
 ENV PDF_STORAGE_PATH=
+ENV DISABLE_ORGANIZATION=false
```

Comment:
I think ENABLE_ORGANIZATION=true/false would be easier ? Default would be true and you could set it to false in the environment to disable it.